gr-utils/python/modtool/core/add.py

```diff
@@ -143,42 +143,49 @@ def run(self):
 
     def _run_cc_qa(self):
         " Add C++ QA files for 3.7 API if intructed from _run_lib"
-        fname_qa_h  = 'qa_{}.h'.format(self.info['blockname'])
-        fname_qa_cc = 'qa_{}.cc'.format(self.info['blockname'])
+        blockname_=self.info['blockname']
+        fname_qa_h  = f'qa_{blockname_}.h'
+        fname_qa_cc = f'qa_{blockname_}.cc'
         self._write_tpl('qa_cpp', 'lib', fname_qa_cc)
         self._write_tpl('qa_h',   'lib', fname_qa_h)
+        modname_= self.info['modname']
+        blockn_=self.info['blockname']
```

Comment:
Why both blockname_ and blockn_.  It would be cleaner just to stick with the dict lookups instead of the intermediate variables

Reply:
I'm OK with the these since they are used multiple times. `blockname_` is the same as `blockn_` ... yes? If so, then you can change the latter to the former & remove the former's definition. Either way will work, of course ...

Reply:
@mormj @michaelld I will optimise this but actually at some places value was changing. So, to avoid wrong values being assigned in strings, I redeclared these multiple times.